static/_app/tools/tradecraft/index.html

```
@@ -0,0 +1,21 @@
+
+<head>
```

HTMLHint (1.5.0)

[error] Doctype must be declared before any non-comment content.

(doctype-first)